receiver/googlecloudmonitoringreceiver/config.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package googlecloudmonitoringreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudmonitoringreceiver"
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+const minCollectionIntervalSeconds = 60
+
+type Config struct {
+	scraperhelper.ControllerConfig `mapstructure:",squash"`
+
+	Region            string    `mapstructure:"region"`
+	ProjectID         string    `mapstructure:"project_id"`
+	ServiceAccountKey string    `mapstructure:"service_account_key"`
+	Services          []Service `mapstructure:"services"`
```

Comment:
A Service, as defined below, only has a single metric.  I would consider renaming this `metrics`.

Reply:
Updated